kubos/test/test_update.py

```diff
@@ -0,0 +1,152 @@
+# Kubos CLI
+# Copyright (C) 2017 Kubos Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from mock import patch, MagicMock, call
+from pip.utils import get_installed_version
+
+from kubos import update
+from kubos.utils.constants import *
+
+class Args():
+    def __init__(self):
+        self.component = None
+
+class UpdateTest(unittest.TestCase):
+    def test_add_options(self):

```

Comment:
Same comment on not testing the  add options function. #34 has already changed the options added to the parser 😄